example-junit/pom.xml

```diff
@@ -14,9 +14,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.epam.reportportal</groupId>
+            <groupId>com.github.reportportal</groupId>
             <artifactId>agent-java-junit</artifactId>
-            <version>5.0.0-BETA-3</version>
+            <version>4ffd0d3d</version>
```

Comment:
Guess that too outdated, please update dependencies.

Reply:
updated